### DISCUSSION TITLE
[Geomap] Support WMTS Provider

### DISCUSSION THREAD

**Discussion by syncush** [grafana/grafana#65326] [Dashboards, Panels & Visualizations]
Hi, 
as of today, Grafana supports XYZ protocol, it would be awesome if I could configure an imagery layer from a WMTS service.

https://openlayers.org/en/latest/examples/wmts.html

I want to be able to provide Geomap with the GetCapabilities URL and the active layer I would like to view.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82775. Feel free to continue the discussion around this there. Thank you!
